suite-native/transactions/src/screens/TransactionDetailScreen.tsx

```diff
@@ -23,16 +25,27 @@ export const TransactionDetailScreen = ({
     const { txid } = route.params;
     const transaction = useSelector(selectTransactionByTxid(txid));
     const blockchain = useSelector(selectBlockchainState);
+    const fiatCurrency = useSelector(selectFiatCurrency);
+    const { FiatAmountFormatter, CryptoAmountFormatter, CurrencySymbolFormatter } = useFormatters();
 
     // TODO please add empty state when design is ready
     if (!transaction) return null;
 
     const confirmations = getConfirmations(transaction, blockchain[transaction.symbol].blockHeight);
-    const totalAmount = formatNetworkAmount(transaction.amount, transaction.symbol, true);
+
+    const transactionAmount = formatNetworkAmount(transaction.amount, transaction.symbol);
+    const fiatAmount = toFiatCurrency(transactionAmount, fiatCurrency.label, transaction.rates);
+    const cryptoAmountFormatted = `${CryptoAmountFormatter.format(transactionAmount, {
+        symbol: transaction.symbol,
+    })} ${CurrencySymbolFormatter.format(transaction.symbol)}`;
 
     return (
         <Screen header={<ScreenHeader />}>
-            <TransactionDetailHeader type={transaction.type} amount={totalAmount} />
+            <TransactionDetailHeader
+                type={transaction.type}
+                amount={cryptoAmountFormatted}
+                fiatAmount={FiatAmountFormatter.format(fiatAmount ?? 0)}
```

Comment:
```suggestion
                fiatAmount={fiatAmount}
```

And format it with FiatAmountFormatter component as I described in another comment https://github.com/trezor/trezor-suite/pull/6568/files#r996975776.

Reply:
[0b65f2d](https://github.com/trezor/trezor-suite/pull/6568/commits/0b65f2d08c3530a7ca8638c400363bd77222c07c)